src/test/java/Homework14.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework14 {
+
+    //Create a test case registrationNavigation() using @Test annotation
+    @Test
+    public static void registrationNavigation () throws InterruptedException {
```

Comment:
It `throws InterruptedException` can be deleted because it is not used 

For example

![image](https://user-images.githubusercontent.com/104761482/200301413-7d0ce3d8-df01-48ef-a40e-a7bffd514cf5.png)